package.json

```diff
@@ -8,9 +8,7 @@
     "postcss-plugin",
     "css-modules"
   ],
-  "files": [
```

Comment:
Looks like index.js is a bit redundant, but let's see it as ability to run not compiled code.